<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZarzadzaniePEWidget</class>
 <widget class="QWidget" name="ZarzadzaniePEWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_zakres">
     <item>
      <widget class="QLabel" name="label_zakres">
       <property name="text">
        <string>Wybierz zakres:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="zakres_combo_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_button">
       <property name="toolTip">
        <string>Odśwież listę zakresów</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/options_icons/options_icon_refresh.png</normaloff>:/icons/options_icons/options_icon_refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_identyfikacja_prg">
       <attribute name="title">
        <string>Identyfikacja PRG</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_tab_prg">
        <item>
         <widget class="QGroupBox" name="groupBox_warstwy">
          <property name="title">
           <string>Wybór warstw</string>
          </property>
          <layout class="QFormLayout" name="formLayout_warstwy">
           <item row="0" column="0">
            <widget class="QLabel" name="label_warstwa_zrodlowa">
             <property name="text">
              <string>Warstwa źródłowa (z danymi PRG):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="source_layer_combobox"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_warstwa_docelowa">
             <property name="text">
              <string>Warstwa docelowa (do uzupełnienia):</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="target_layer_combobox"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_mapowanie">
          <property name="title">
           <string>Mapowanie atrybutów</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_mapowanie">
           <item row="0" column="1">
            <widget class="QLabel" name="label_arrow1">
             <property name="text">
              <string>=&gt;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="target_attr_combobox_2"/>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="source_attr_combobox_1"/>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_arrow2">
             <property name="text">
              <string>=&gt;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="target_attr_combobox_1"/>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="source_attr_combobox_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QComboBox" name="source_attr_combobox_3"/>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_arrow3">
             <property name="text">
              <string>=&gt;</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="target_attr_combobox_3"/>
           </item>
           <item row="3" column="0">
            <widget class="QComboBox" name="source_attr_combobox_4"/>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_arrow4">
             <property name="text">
              <string>=&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QComboBox" name="target_attr_combobox_4"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_opcje">
          <property name="title">
           <string>Opcje</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_opcje">
           <item>
            <widget class="QCheckBox" name="ulic_checkbox">
             <property name="text">
              <string>Uzupełnij atrybut ULIC wartością 99999, jeśli jest pusty</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_strategia">
             <property name="title">
              <string>Strategia aktualizacji</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_strategia">
              <item>
               <widget class="QRadioButton" name="nadpisz_radio">
                <property name="text">
                 <string>Nadpisz istniejące wartości</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="pomijaj_radio">
                <property name="text">
                 <string>Pomijaj istniejące wartości</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_zasieg">
             <property name="title">
              <string>Zasięg działania</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_zasieg">
              <item>
               <widget class="QLabel" name="label_zasieg">
                <property name="text">
                 <string>Określ odległość graniczną [m]:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="distance_spinbox">
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_crs_warning">
          <property name="text">
           <string>Uwaga! W przypadku warstw z układem współrzędnych innych niż metryczne zostanie wykonane przeliczenie &quot;w locie&quot; na potrzeby określenia wskazanego zasięgu.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_wspolrzedne">
       <attribute name="title">
        <string>Współrzędne obiektów PE</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_tab_wspolrzedne">
        <item>
         <widget class="QGroupBox" name="groupBox_wybor_warstwy_wspolrzedne">
          <property name="title">
           <string>Wybór warstwy</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_wybor_warstwy">
           <item>
            <layout class="QFormLayout" name="formLayout_wybor_warstwy">
             <item row="0" column="0">
              <widget class="QLabel" name="label_wybor_warstwy_wspolrzedne">
               <property name="text">
                <string>Wybierz warstwę:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="wspolrzedne_layer_combobox"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="wspolrzedne_show_all_layers_checkbox">
             <property name="text">
              <string>Wyświetl na liście wszystkie istniejące warstwy</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_strategia_wspolrzedne">
          <property name="title">
           <string>Strategia aktualizacji</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_strategia_wspolrzedne">
           <item>
            <widget class="QRadioButton" name="nadpisz_radio_wspolrzedne">
             <property name="text">
              <string>Nadpisz istniejące wartości</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="pomijaj_radio_wspolrzedne">
             <property name="text">
              <string>Pomijaj istniejące wartości</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_wspolrzedne">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="output_widget_placeholder" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>